code/game/gamemodes/dynamic/dynamic.dm

```diff
@@ -0,0 +1,730 @@
+			else
+				. += "<b>Core Territory</b></center><BR>"
+				. += "Your station orbits within reliably mundane, secure space. Although Nanotrasen has a firm grip on security in your region, the valuable resources and strategic position aboard your station make it a potential target for infiltrations. Monitor crew for non-loyal behavior, but expect a relatively tame shift free of large-scale destruction. We expect great things from your station."
+		if(20 to 39)
+			. += "<b>Anomalous Exogeology</b></center><BR>"
+			. += "Although your station lies within what is generally considered Nanotrasen-controlled space, the course of its orbit has caused it to cross unusually close to exogeological features with anomalous readings. Although these features offer opportunities for our research department, it is known that these little understood readings are often correlated with increased activity from competing interstellar organizations and individuals, among them the Wizard Federation and Cult of the Geometer of Blood - all known competitors for Anomaly Type B sites. Exercise elevated caution."
+		if(40 to 65)
+			. += "<b>Contested System</b></center><BR>"
+			. += "Your station's orbit passes along the edge of Nanotrasen's sphere of influence. While subversive elements remain the most likely threat against your station, hostile organizations are bolder here, where our grip is weaker. Exercise increased caution against elite Syndicate strike forces, or Executives forbid, some kind of ill-conceived unionizing attempt."
+		if(66 to 79)
+			. += "<b>Uncharted Space</b></center><BR>"
+			. += "Congratulations and thank you for participating in the NT 'Frontier' space program! Your station is actively orbiting a high value system far from the nearest support stations. Little is known about your region of space, and the opportunity to encounter the unknown invites greater glory. You are encouraged to elevate security as necessary to protect Nanotrasen assets."
+		if(80 to 99)
+			. += "<b>Black Orbit</b></center><BR>"
+			. += "As part of a mandatory security protocol, we are required to inform you that as a result of your orbital pattern directly behind an astrological body (oriented from our nearest observatory), your station will be under decreased monitoring and support. It is anticipated that your extreme location and decreased surveillance could pose security risks. Avoid unnecessary risks and attempt to keep your station in one piece."
+		if(100)
+			. += "<b>Impending Doom</b></center><BR>"
+			. += "Your station is somehow in the middle of hostile territory, in clear view of any enemy of the corporation. Your likelihood to survive is low, and station destruction is expected and almost inevitable. Secure any sensitive material and neutralize any enemy you will come across. It is important that you at least try to maintain the station.<BR>"

```

Comment:
Make the centcomm roundstart report that generates on paper into an automated announcement, similar to code blue ert, depending on the threat level

Reply:
I really have no idea on how to do this, but I'll try and find the code blue ERT announcement, and put that in, in some way.